src/models/media.ts

```diff
@@ -21,7 +21,7 @@ export interface Media {
 
   license: License
   license_version: LicenseVersion
-  license_url: string
+  license_url?: string
```

Comment:
Why does the `licensr_url` change to an optional field?

Reply:
Practically it's never undefined, I added that just to ensure that things work even if it was undefined.